content/en/service_owners_guide/plan.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Plan your Datadog implementation
+description: Setup and implement Datadog as a Service owner to avoid pit-falls down the road
+further_reading:
+- link: "/getting_started/tagging/unified_service_tagging/"
+  tag: "Documentation"
+  text: "Getting Started with Unified Service Tagging"
+- link: "/getting_started/tracing/"
+  tag: "Documentation"
+  text: "Getting Started with APM Tracing"
+---
+
+## Design
+
+### Sizing exercise
+
+Defining a clear end goal is essential when starting a significant product implementation. However, it's impractical to know all requirements from the beginning. Product engineers iterate deployments, and systems operations manage changes to mitigate risk. Similarly, a large-scale Datadog deployment benefits from standard project management practices. Certain Datadog elements should be included in this process. Survey outlines are useful for assessing and planning your needs.
+
+A sample survey form might look like this: 
```

Comment:
Perhaps move the sample survey form after the recommendation

Reply:
We might want to make this look more like an intake form/ diagram - more specific to a working example, possibly a graphic.
if it's not ready we can remove for this iteration.

Reply:
for not adjusted the formatting on this and moved it below the recommendation